src/testing/loadtest-azure/scripts/file-upload-to-loadtest-harness.ps1

```diff
@@ -0,0 +1,9 @@
+# file-upload-to-loadtest-harness.ps1
+$loadTestId = "6ac85c38-e783-4ec7-94cc-db6c68fef1aa"
+$testFileName = "/mnt/c/git/AlwaysOn-foundational/src/testing/loadtest-azure/scripts/catalogue-test.jmx"
+$apiEndpoint = "5e241573-3a5f-4361-bf53-1ae7bde73cb7.neu.cnt-prod.loadtesting.azure.com" #needs to be updated... will be returned when resource is created
```

Comment:
cleanup?

Reply:
Cleaned them up a bit. I'd like to get rid of all *-harness.ps1 at some point. But at the moment the add a lot of value.

Reply:
what are you still using them for?

Reply:
Let's drop them. I re-worked most of the wrapper scripts and they're pretty easy to use now. That should be sufficient and reduce the need for the harness files.

Reply:
Dropped